src/Installers/Windows/Directory.Build.props

```diff
@@ -9,6 +9,9 @@
       Read more here => https://github.com/dotnet/project-system/blob/main/docs/build-acceleration.md#limitations
     -->
     <AccelerateBuildsInVisualStudio>false</AccelerateBuildsInVisualStudio>
+
+    <!-- Suppress validation to avoid Applocker failures when running light.exe on dev machines. -->
+    <SuppressValidation>true</SuppressValidation>
```

Comment:
Do we need this for the .sfxproj's in https://github.com/dotnet/aspnetcore/tree/main/src/Framework as well? I think this d.b.p is only used by the Hosting Bundle

Reply:
We only need it for the wixprojs as far as I can tell. I think only ANCM, but I put it here since ANCM and the hosting bundle projects import this file